### ISSUE TITLE
Test run times don't make sense

### ISSUE THREAD

**Issue by doerwalter** [christiansandberg/pytest-reporter#4]
When running tests with `pytest-xdist` the test run times don't make sense, as can be seen in the following screen shot.

![Bildschirmfoto 2020-08-17 um 16 44 01](https://user-images.githubusercontent.com/1387028/90409380-5c3fc780-e0a9-11ea-8e03-bd917102f5ef.png)

The test was run with `pytest -v -l -n10` and the report states a total run time of 36 minutes. However for some of the individual test functions a longer run time is reported. For `test_orasql.py` and `test_ul4.py` a run time of over two hours is reported.

**Comment by christiansandberg** [comment]
Yes, that's another thing that might need adjusting when it comes to pytest-xdist. The time displayed is the sum of all test durations which may or may not be what you want. On one hand it doesn't make sense since the actually time to run the tests might be shorter if they are executed in parallell, on the other hand it is probably the best indicator if you want to know which modules take the longest time to execute. If it would be the wall clock time it might vary a lot depending on how the scheduler works. You could also run tests in random order in which case it would be even more misleading.

So I don't really know what to display there. Maybe it is best to just remove it to avoid confusion...
